### DISCUSSION TITLE
Alpine Mask v 3.10.2 -> 3.12.0

### DISCUSSION THREAD

**Discussion by gormmo** [alpinejs/alpine#3456] [7. Bugs (archived)] [answered]
Hi.
When updating the mask plugin i get this error after build. I'm using laravel-mix v.6.0.27.

![image](https://user-images.githubusercontent.com/6972592/224360496-67502936-184a-43d7-af84-c1bf2ce6ad0e.png)

**Comment by SimoTod** [comment, marked as answer]
you need to keep alpine and the plugins on the same version

**Reply by gormmo** [reply]
Ok, great. Thanks for your answer :-) Did not know that.

**Reply by SimoTod** [reply]
It's not always the case but sometimes there are changes in one package that affect the other one. In general, it's a good idea to keep them on the same version

**Reply by ekwoka** [reply]
Yup, it's the reason why they generally step together. So you can always know that if they have the same version number, they are compatible. any other combination may not be.
